libopae/plugins/ase/scripts/generate_ase_environment.py

```diff
@@ -60,8 +60,8 @@
 from sets import Set
 import shutil
 
-reload(sys)
-sys.setdefaultencoding('utf8')
+# reload(sys)
+# sys.setdefaultencoding('utf8')
```

Comment:
Need this enabled?

Reply:
it might be needed in some environment. For now I will remove it.